api/maven-api-core/src/main/java/org/apache/maven/api/services/MessageBuilder.java

```diff
@@ -206,4 +206,11 @@ public interface MessageBuilder {
      */
     @Nonnull
     String build();
+
+    /**
+     * Set the buffer length.
+     *
+     * @param length the new length
+     */
+    void setLength(int length);
```

Comment:
Shouldn't this be reused in #1269 somehow ?  I mean that PR mixes ansi formatting with string builders...  No big deal for now though.

Reply:
For me it is not related, the usage of the `MessageUtils` class is not the same